api/v1alpha08/kogitoserverlessworkflow_types.go

```diff
@@ -391,25 +390,36 @@ type Endpoint struct {
 
 // KogitoServerlessWorkflowStatus defines the observed state of KogitoServerlessWorkflow
 type KogitoServerlessWorkflowStatus struct {
-	Conditions  BuildStatusCondition         `json:"conditions,omitempty"`
-	Endpoints   []Endpoint                   `json:"endpoints"`
-	Address     duckv1.Addressable           `json:"address,omitempty"`
-	Deployments olm.DeploymentStatus         `json:"deployments"`
-	Phase       ConditionType                `json:"phase,omitempty"`
-	Applied     KogitoServerlessWorkflowSpec `json:"applied,omitempty"`
-	Version     string                       `json:"version,omitempty"`
+	Endpoints []Endpoint                   `json:"endpoints,omitempty"`
+	Address   duckv1.Addressable           `json:"address,omitempty"`
+	Condition ConditionType                `json:"phase,omitempty"`
```

Comment:
is the json property name correct?

Reply:
No, it should be condition, I'll correct it.